# **SUTD 10.006 Chemistry and Biology: The Natural World 2019 Progressive Learning Worksheet Answers**

### **Week 1**

1. C

2. a) 1s<sup>2</sup> 2s<sup>2</sup> 2p<sup>6</sup> 3s<sup>2</sup> 3p<sup>6</sup> 3d<sup>5</sup> 4s<sup>1</sup>  **OR** [Ar] 3d<sup>5</sup> 4s<sup>1</sup>

   b) 1s<sup>2</sup> 2s<sup>2</sup> 2p<sup>6</sup> 3s<sup>2</sup> 3p<sup>6</sup> 3d<sup>4</sup>  **OR** [Ar] 3d<sup>4</sup>
   
3. D

4. C

5. C

6. A

7. B

8. B

9. B

10. A

11. C

12. D

13. 1.16 × 10<sup>16</sup> Cr atoms

14. 3NO<sub>2</sub> + H<sub>2</sub>O → 2HNO<sub>3</sub> + NO

    8.21 g

15. A

### **Week 2**

1. D

2. C

3. B

4. A

| Compound |                     Best Lewis Structure                     | Steric Number |   Molecular Shape    |  Bond Angle(s)   |
| :------: | :----------------------------------------------------------: | :-----------: | :------------------: | :--------------: |
|  BF<sub>3</sub>   | <img src=".\assets\Lewis_Structures\BF3.png" width="200px" style="height:200px"/> |       3       |   Trigonal Planar    |       120°       |
|  CCl<sub>4</sub>  | <img src=".\assets\Lewis_Structures\CCl4.png" width="200px" style="height:200px"/> |       4       |     Tetrahedral      |      109.5°      |
|  PBr<sub>3</sub>  | <img src=".\assets\Lewis_Structures\PBr3.jpg" width="200px" style="height:200px"/> |       4       |  Trigonal Pyramidal  |       107°       |
|  PCl<sub>5</sub>  | <img src=".\assets\Lewis_Structures\PCl5.png" width="200px" style="height:200px"/> |       5       | Trigonal Bipyramidal |   90° and 120°   |
|  SCl<sub>4</sub>  | <img src=".\assets\Lewis_Structures\SCl4.jpg" width="200px" style="height:200px"/> |       5       |       See-saw        |  <90° and <120°  |
|  ICl<sub>3</sub>  | <img src=".\assets\Lewis_Structures\ICl3.png" width="200px" style="height:200px"/> |       5       |  T-shaped   | 87.5° (and 180°) |

5. B

6. a) 888 kJ/mol

   b) 194 kJ/mol (neglecting the small short-range repulsion between the ion cores of Li<sup>+</sup> and F<sup>-</sup>)

   c) 7.15 Å

7. G

8. a) London dispersion forces, dipole-dipole interactions and hydrogen bonding.

   b) London dispersion forces and dipole-dipole interactions.

   c) London dispersion forces.

9. A

10. C

11. A

12. C

### **Week 3**

1. D

2. A

3. B

4. D

5. <img src=".\assets\Week_3_Q5_abc.jpg" width="500px" style="height:300px"/> 

   (For a) and b), choose one circle and one triangle respectively. For c), there is only one rectangle. I apologize for the awful shape drawings.)

   d) They both have carboxyl and amine groups, with 4 atoms bonded directly to the α-carbon.

   e) They have different R-groups.

6. A

7. a) <img src=".\assets\Amino_Acids_Interactions\Cysteine-Cysteine.png" width="250px" style="height:300px"/>

   ​	Disulfide bonds.

   b) <img src=".\assets\Amino_Acids_Interactions\Isoleucine-Valine.png" width="360px" style="height:200px"/>

   ​	Van der Waals interactions.

   c) <img src=".\assets\Amino_Acids_Interactions\Glutamic_Acid-Lysine.png" width="250px" style="height:300px"/>

   ​	Ion-ion interactions.

   d) <img src=".\assets\Amino_Acids_Interactions\Serine-Glutamine.png" width="500px" style="height:220px"/>

   ​	Hydrogen bonds.

### **Week 4**

1. D

2. D

3. a) 2.52

   b) No, the reaction is not at equilibrium. As Q > K<sub>c</sub>, the reaction will shift to the left to reach equilibrium.

4. 605

5. C

6. B

7. C

8. [CO] = 1.27 M

   [H<sub>2</sub>O] = 0.269 M

   [CO<sub>2</sub>] = 0.731 M

   [H<sub>2</sub>] = 0.731 M

9. [COCl<sub>2</sub>] = 0.480 M

   [CO] = 0.0200 M

   [Cl<sub>2</sub>] = 0.0200 M

10. 300 K

11. a) −39.5 kJ/mol

    b) −79.0 kJ/mol

    c) 2.025 × 10<sup>13</sup>

    ​	As the K<sub>A</sub> value is very large (larger than for the monomer-ligand interaction), the binding affinity is very high for the dimer-ligand interaction (higher than the affinity for the monomer-ligand interaction).

12. C

13. A

14. C

15. C

16. A

17. C

18. −1194 kJ/mol

### **Week 5**

1. A

2. D

3. B

4. B

5. A

6. B

7. D

8. [H<sub>3</sub>O<sup>+</sup>] = 5.46 × 10<sup>−4</sup> M

   pH = 3.26

9. 3.37

10. a) 10.1

    b) 5.89

    c) 2.26

11. C

12. Li(*s*)

13. F<sub>2</sub>(*g*)

14. D

### Week 8

1. D

2. C

3. C

4. C

5. B

6. B

7. D

8. E

9. a) 2

   b) Second step.

   c) Exothermic.

10. a) 272 kJ/mol

    b) 3.34 × 10<sup>-5</sup> s<sup>-1</sup>

11. a) 2NO(g) + Br<sub>2</sub>(g) → 2NOBr(g)

    b) NOBr<sub>2</sub>(g)

    c) Rate = <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q11c_part1.gif" style="max-width:100%">, where <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q11c_part2.gif" style="max-width:100%">.

    ​	Yes, it is consistent with the actual experimental rate law.

12. Rate = <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q12.gif" style="max-width:100%">

    The proposed reaction mechanism is not acceptable since it is not consistent the actual experimental rate law.

13. a) Rate = <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q13a.gif" style="max-width:100%">

    b) It is not exactly consistent with the actual experimental rate law.

    c) (i) Rate = <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q13c_i.gif" style="max-width:100%">
       (ii) Rate = <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q13c_ii.gif" style="max-width:100%">

    ​		Only the case with low oxygen concentration is consistent with the actual experimental rate law, with <img src=".\assets\Kinetics_Rate_Equations\Week_8_Q13c_final.gif" style="max-width:100%">.

### Week 9

1. E

2. C

3. B

4. E

5. A

6. D

7. A

8. E

9. B

10. B

11. C

12. C

13. C

14. - Both are pathways for producing ATP by harvesting the chemical energy of food.
    - Both processes use glycolysis to oxidise glucose and other organic molecules to pyruvate, with a net production of 2 ATP by substrate-level phosphorylation.
    - In both processes, NAD<sup>+</sup> is the oxidising agent that accepts electrons from food during glycolysis.
    - In fermentation, there is only glycolysis (there is no citric acid cycle, electron transport chain and chemiosmosis) unlike in aerobic respiration.
    - In fermentation, the NAD<sup>+</sup> is regenerated from the conversion of pyruvate to lactate. In aerobic respiration, the NAD<sup>+</sup> is regenerated after donating the proton/electron that starts the electron transport chain.
    - Aerobic produces more ATP than fermentation.

15. CO<sub>2</sub> is released from the pyruvate, which is the end product of glycolysis, through pyruvate oxidation. CO<sub>2</sub> is also released during the citric acid cycle.

16. The proton-motive force (movement of H<sup>+</sup> down its concentration gradient from intermembrane space into the matrix) rotates the F<sub>0</sub> region of the ATP synthase situated in the mitochondria's inner membrane, specifically the c-subunits. The rotating c-subunits cause the γ-subunit (the internal rod) to spin, and this induces a conformational change in the β-subunit (in the F<sub>1</sub> region), which activates the binding of ADP and P<sub>i</sub> to produce ATP.

### Week 10

1. B

2. C

3. C

4. A

5. B

6. B

7. A

8. B

9. - Helicase: It unzips the DNA helix ahead of the replication fork. It uses ATP hydrolysis to move forward along the DNA strand.
   - Primase: It synthesises RNA primers (5-10 bond pairs) using parent DNA as template. DNA extension starts at 3’-OH end of the primer.
   - DNA Polymerase: It adds the next nucleotide to the 3’-OH end of an existing chain in the 5’-to-3’ direction.
   - Nuclease: It catalyses the cleavage of phosphodiester bonds between nucleotides of nucleic acids. It helps with proofreading and DNA mismatch repair as it is able to recognise and remove DNA oxidation damage.
   - Repair Polymerase: It replaces RNA primers with DNA nucleotides.
   - Ligase: It seals the nicks between Okazaki fragments on the lagging strand.
   - Topoisomerase (Type II): It changes the DNA topology by breaking, swivelling and rejoining double-stranded DNA molecule. These enzymes can introduce or remove supercoils and can separate two DNA duplexes that are intertwined. The purpose is to reduce the tension in DNA.

10. A. (iv) 5’-UGUUCGUAGA-3’. Replace the T-s in 5’-TGTTCGTAGA-3’ with U-s as primer is made of RNA.

    B. In the 5’ → 3’ direction.

    C. In the 3’ → 5’ direction.

11. <img src=".\assets\Week_10_Q11_abcd.jpg" style="max-width:100%"/>

### Week 12

1. C

2. B

3. D

4. A

5. C

6. A

7. B

8. B

9. A

10. C

11. B

12. A

13. A

14. A

15. Firstly, the RNA molecule should have uracil instead of thymine bases. Secondly, the polarity of the molecule is incorrectly labelled. The correct RNA molecule produced, using the bottom strand of the DNA duplex as a template, would be 5’-GGCAUGGCAAUAUUGUAGUA-3’.

16. 5’-CUUCCAUCCA...-3’

17. - A 5’-cap is added. A modified guanine is added to the 5’-end of mature mRNA. It promotes the exporting of the mRNA from the nucleus, protects the mRNA from degradation by exonucleases and also promotes ribosomal binding.
    - A poly-A tail, comprised of approximately 200 adenine molecules, is added to the 3’-end of the mature mRNA. It provides stability to the mature mRNA and prevents its degradation by exonucleases. It also promotes the exporting of the mRNA from the nucleus to the cytosol.
    - Introns are spliced out. Pre-mRNA comprises of introns and exons. Introns are non-coding sequences and will be removed, leaving only the protein-coding sequences which will be translated to make a polypeptide.

18. - Stage 2: Peptide bond formation. The COOH-end of the growing polypeptide chain is uncoupled from the tRNA at the P site to form a peptide bond with the free -NH<sub>2</sub> group of the next amino acid at the A site.
    - Stage 3: Translocation. The large subunit shifts, moving the 2 tRNA-s into the E and P sites. The small subunit moves exactly 3 nucleotides back to its original position. During this process, one GTP molecule is consumed, while one GDP molecule and one P<sub>i</sub> molecule is produced. Finally, the spent tRNA at the E site is ejected. The ribosome is ready for the next aminoacyl tRNA.

### Week 13

1. C

2. A

3. A

4. B

5. B

6. B

7. B

8. B

9. A

10. D

11. C

12. D

13. D

14. A

15. B

16. D

17. A

18. A) You would first digest your sample with a combination of restriction enzymes selected so that they give a set of fragment sizes that could have come from only one of the plasmids. Then you would run the resulting mixture of DNA fragments on a gel alongside a set of size markers and determine the size of each fragment. By looking at the restriction maps, you should then be able to match your results to one of the plasmids.

    B) Digestion with any of the following combinations will enable you to distinguish which plasmid you have: *Hin*dIII + *Bgl*II; *Eco*RI + *Bgl*II; *Eco*RI + *Bgl*II + *Hin*dIII. The plasmids are the same size, so you cannot distinguish between them simply by making a single cut (with *Hin*dIII) and determining the size of the complete DNA by gel electrophoresis. Nor can you distinguish them by cutting with all four restriction nucleases, because the set of fragment sizes produced from both plasmids will be the same. Cutting with *Bam*H I or *Eco*RI on their own is not sufficient because you will get bands of the same size from both plasmid A and plasmid B. The only difference between the two plasmids is in the location of the *Bgl*II site relative to the two *Bam*H I sites, so if you cut with an enzyme that cuts outside the *Bam*H I fragment and with *Bgl*II, you will get different-sized fragments from the two plasmids.

19. - Situation A: Operon off. CAP will not bind in the presence of glucose.
    - Situation B: Operon off. Although normally the *lac* repressor would bind in the absence of lactose, the lack of the *lac* repressor in this case does not matter because the presence of glucose means that the CAP protein will not bind and activate transcription.
    - Situation C: Operon on. Normally in the absence of both glucose and lactose, the operon would be off. However, because the cells lack the *lac* repressor, the cells cannot sense the absence of lactose. Because the CAP protein will bind and activate transcription, the operon will be on.
    - Situation D: Operon on. The CAP protein will bind and activate transcription because of the presence of glucose. It does not matter whether the *lac* repressor gene is mutant, because there is lactose available.
